### DISCUSSION TITLE
Discuss: Modularization of Components

### DISCUSSION THREAD

**Discussion by birm** [camicroscope/GSOC#51] [Ideas]
Discuss, or Question and answer about this project specifically.

**Comment by Jyotiraditya7** [comment]
Hey, I'm interested in this project idea as I'm familiar with the tech stack required . By modularization of frontend components, do you mean using modular tools like React for the front-end development part ? Also it would be really helpful if you could specify the expected outcome/functionality of the modified components .

**Comment by Apurv428** [comment]
Hi @birm , this project is not mentioned in this [link](https://github.com/camicroscope/GSOC). So, will this be a part of idea list or not?

**Reply by birm** [reply]
We're deciding if we want to keep this one as a GSOC project. It requires much more familiarity with the codebase than the others.

**Reply by Apurv428** [reply]
Okay, Thanks for the reply!

**Comment by birm** [comment]
We have decided not to promote this as a GSOC project this year. As earlier mentioned, it requires a specific familiarity with caMicroscope's code, and honestly probably isn't all too interesting. It's basically a refactor.
